### DISCUSSION TITLE
Proposal: Adjusting Mainboard Temperature Limit to achieve max 50 Degrees for Creality K1 Series

### DISCUSSION THREAD

**Discussion by EonPrintCrafter** [Guilouz/Creality-Helper-Script-Wiki#607] [Features Request]
Hi Guilouz,

I hope you're doing well. I wanted to reach out regarding a potential enhancement for the Creality Helper Script, which has been incredibly useful for many of us in the Creality K1 series printer community.

I'm proposing a simple modification to the fans-control.cfg file within the script. Specifically, I suggest adjusting the target temperature from 50.0 to 43.0 in the [temperature_fan soc_fan] section. This change aims to keep the mainboard temperature below 50 degrees Celsius during operation, contributing to improved reliability and longevity of the printers.

Here's the requested modification:

[temperature_fan soc_fan]
pin: PB2
cycle_time: 0.0100
hardware_pwm: false
max_power: 1
shutdown_speed: 0
sensor_type: temperature_mcu
min_temp: 0
max_temp: 100
control: watermark
max_delta: 2
target_temp: 43.0
max_speed: 1.0
min_speed: 0.0
tachometer_pin:PC6

Implementing this change would ensure cooler operating conditions for the mainboard without compromising performance and maybe even gain some very little.

Could you please consider integrating this adjustment into the official Creality Helper Script repository?

Thanks for your consideration.

**Comment by TheWebMachine** [comment]
Actually, this presents an interesting question:

Is the MCU fan being controlled with varying speed based on temp? Or is it just on 100% above the target?  I don't think I've ever heard it run at anything less than full throat from day one.

I kinda hate how loud the MCU fan is and would rather see a menu in the Helper script that let's us set the temp *and* max fan speed (especially if a binary on/off). 

I suppose I should just pop the bottom off and replace the MCU fan with a Noctua, like I do with every other noisy fan I encounter. 🤣 I hate cheap fan noise and it kinda echos from under the K1C's bottom compartment, which is even worse.

**Comment by Guilouz** [comment]
Done. I have set it to 45C° like ender-3 v3.

**Reply by EonPrintCrafter** [reply]
Hi Guilouz,

Thank you for addressing the modification suggestion so quickly. Setting the target temperature to 45°C is a great step. This adjustment should help maintain optimal operating conditions for the mainboard.

Additionally, I appreciate the swift integration of this change into the Creality Helper Script repository. This will undoubtedly contribute to the longevity and reliability of our printers.

Thanks again for your prompt response and support.
